DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -514,6 +514,20 @@
   <class name="edm::Wrapper< std::vector<edm::Ptr<pat::Jet> > >" />
   <class name="edm::Wrapper< std::vector< std::vector<edm::Ptr<pat::Jet> > > >" />
 
+  <class name="pat::PATTauDiscriminatorBase">
+    <field name="transientVector_" transient="true"/>
+  </class>  
+  <class name="pat::PATTauDiscriminator"  ClassVersion="3">
+   <version ClassVersion="3" checksum="2916098408"/>  
+  </class>  
```

Comment:
I think to handle a version for a template one can't use the classes_def.xml. Instead, one has to add the Class_Version function to the class body itself. E.g.
https://github.com/cms-sw/cmssw/blob/0397259dd747cee94b68928f17976224c037057a/DataFormats/Provenance/interface/Hash.h#L80